src/generic_core/uproxy_core.ts

```diff
@@ -716,6 +723,56 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
   public updateOrgPolicy(policy :Object) :void {
     /* TODO(xwsxethan) handle updating org policy */
   }
+
+  public cloudDestroy = (providerName :string) :Promise<void> =>  {
+    log.info('cloudDestroy', providerName, DEFAULT_SERVER_NAME);
+
+    const provisioner = freedom[CLOUD_PROVIDER_MODULE_PREFIX + providerName]();
```

Comment:
Suggest you factor out the provisioner module name into a `const`, since you need it again in a few lines later.